src/Core/Response.php

```diff
@@ -45,21 +45,27 @@ public function getErrorCode() : int
 	}
 
 	/**
-	 * Fetches the Error Message From Response
+	 * Fetches the Error Message (including Extended Error Info) From Response
 	 * @method getError
 	 * @author PA
 	 * @date   2017-01-10
 	 * @return string     Error Message
 	 */
 	public function getError() : string
 	{
-		return (string) $this->response->ErrDesc;
+	    $extendedErrorInfo = isset($this->response->ExtendedErrorInfo) ? (string) $this->response->ExtendedErrorInfo : '';
+	    $error = isset($this->response->ErrDesc) ? (string) $this->response->ErrDesc : '';
```

Comment:
1. $this->response->ErrDesc always exists when the function is called. so it does not need to check for isset.

If it doesn't exist we should be throwing an exception here, because the ErrDesc is not present when it should be.